partiql-ast/src/main/java/org/partiql/ast/v1/expr/ExprVar.java

```diff
@@ -0,0 +1,48 @@
+package org.partiql.ast.v1.expr;
+
+import org.jetbrains.annotations.NotNull;
+import org.partiql.ast.v1.AstNode;
+import org.partiql.ast.v1.AstVisitor;
+import org.partiql.ast.v1.Identifier;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * TODO docs, equals, hashcode
+ */
+public class ExprVar extends Expr {
+    @NotNull
+    public Identifier identifier;
```

Comment:
```suggestion
    @NotNull
    public IdentifierChain identifier;
```

This would more closely align with the EBNF. Essentially all areas (except for projection aliases, `AS <identifier>`) uses `<identifier chain>` or `<local or schema qualified name>`. That being said, the chain may only have the root.

Up to you on this. It could help our modeling since we would never have to switch on the variants. You could have two distinct classes that don't derive from the same abstract class.